collector/nps.go

```diff
@@ -0,0 +1,422 @@
+package collector
+
+import (
+	"github.com/prometheus-community/windows_exporter/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/yusufpapurcu/wmi"
+)
+
+func init() {
+	registerCollector("nps", newnpsCollector) // TODO: Add any perflib dependencies here
+}
```

Comment:
The `init` function here will need to be removed, with the collector being registered in the [init.go](https://github.com/prometheus-community/windows_exporter/blob/b033ff71ffbd85e9774a9bdcc25c9cd91d1e5e8e/collector/init.go#L33) file.